packages/cozy-harvest-lib/src/services/budget-insight.js

```diff
@@ -215,10 +215,6 @@ export const setBIConnectionId = (originalAccount, biConnectionId) => {
   return account
 }
 
-export const getBIConnectionId = account => {
-  return get(account, 'data.auth.bi.connId')
-}
```

Comment:
Good question, this error is not caught. We are in the context of the "blank page" in the popup or InAppBrowser and we have at the moment no way to display an error message to the user.

Not that this case is not supposed to happen since we already got in code using bankId to have this window/InAppBrowser displayed (getTemporaryToken).

What should I do ? Replace this thrown error with an error log ?